rpmio/rpmpgp_internal.c

```diff
@@ -1150,9 +1153,13 @@ rpmRC pgpVerifySignature(pgpDigParams key, pgpDigParams sig, DIGEST_CTX hashctx)
      * done all we can, return NOKEY to indicate "looks okay but dunno."
      */
     if (key && key->alg) {
+	if (key->tag != PGPTAG_PUBLIC_KEY &&
+	    key->tag != PGPTAG_PUBLIC_SUBKEY)
+	    goto exit; /* not a public key */
```

Comment:
Indent level is broken, as in difficult to read. With an isKey() helper, that goes away. And the comment becomes even more redundant than it is already.